app/frontend_messages.py

```diff
@@ -0,0 +1,4 @@
+messages = {
+  'MULTIPLE_SURVEYS': "Unfortunately you can only complete one survey at a time.",
```

Comment:
Don't forget to use gettext here so strings can be translated (prefix them with _ etc).